tests/helpers/strings.spec.js

```diff
@@ -1,6 +1,6 @@
 
-import Handlebars from 'handlebars';
 import strings from '../../src/helpers/strings';
+import Handlebars from 'handlebars';
```

Comment:
`import {compile} from 'handlebars';` as all we need to do is just `compile` the templates. 

And you could do just `compile(html)` instead of doing `Handlebars.compile(html)`.